frontend/src/metabase/user/components/LoginHistoryList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+
+import LoginHistory from "metabase/entities/loginHistory";
+
+const LoginHistoryItemRow = item => {
+  return (
+    <tr>
+      <td>
+        <tt>{item.timestamp}</tt>
+      </td>
+      <td>{item.location && item.location.description}</td>
+      <td>
+        <tt>{item.device_description}</tt>
+      </td>
+      <td>{item.active ? "✅" : "❌"}</td>
+    </tr>
+  );
+};
+
+@LoginHistory.loadList()
+export default class LoginHistoryList extends Component {
+  render() {
+    const { loginHistory } = this.props;
+
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Timestamp</th>
```

Comment:
I'm not a native speaker but perhaps for a layperson, "Last accessed", is less jargon-y? This is the term used by e.g. GitHub in their login history screen.